pep0/pep.py

```diff
@@ -62,34 +62,38 @@ class Author(object):
 
     def __init__(self, author_and_email_tuple):
         """Parse the name and email address of an author."""
+        self.first = self.last = ''
+
         name, email = author_and_email_tuple
         self.first_last = name.strip()
         self.email = email.lower()
-        last_name_fragment, suffix = self._last_name(name)
-        name_sep = name.index(last_name_fragment)
-        self.first = name[:name_sep].rstrip()
-        self.last = last_name_fragment
-        if self.last[1] == u'.':
-            # Add an escape to avoid docutils turning `v.` into `22.`.
-            self.last = u'\\' + self.last
-        self.suffix = suffix
-        if not self.first:
-            self.last_first = self.last
+
+        name_dict = self._parse_name(name)
+        self.suffix = name_dict.get("suffix")
+        if name_dict.get("name"):
+            self.last_first = name_dict["name"]
+            self.nick = name_dict["name"]
         else:
-            self.last_first = u', '.join([self.last, self.first])
-            if self.suffix:
-                self.last_first += u', ' + self.suffix
+            self.first = name_dict["forename"].rstrip()
+            self.last = name_dict["surname"]
+            if self.last[1] == ".":
+                # Add an escape to avoid docutils turning `v.` into `22.`.
+                self.last = "\\" + self.last
+            self.last_first = ", ".join([self.last, self.first])
+            self.nick = self.last
+
+        if self.suffix:
+            self.last_first += ", " + self.suffix
+
         if self.last == "van Rossum":
             # Special case for our beloved BDFL. :)
             if self.first == "Guido":
                 self.nick = "GvR"
             elif self.first == "Just":
                 self.nick = "JvR"
             else:
-                raise ValueError("unknown van Rossum %r!" % self)
-            self.last_first += " (%s)" % (self.nick,)
-        else:
-            self.nick = self.last
+                raise ValueError(f"unknown van Rossum {self}!")
```

Comment:
```suggestion
                raise ValueError(f"unknown van Rossum {self!r}!")
```
to keep any space, quotes, etc.

Reply:
Having just tested the Author class doesn't print nicely normally or with the repr flag, so will change to printing the inputted name